source/includes/write/replace.cpp

```diff
@@ -0,0 +1,99 @@
+#include <iostream>
+
+#include <bsoncxx/builder/basic/document.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
+#include <mongocxx/instance.hpp>
+#include <mongocxx/uri.hpp>
+
+using bsoncxx::builder::basic::kvp;
+using bsoncxx::builder::basic::make_document;
+
+int main() {
+    mongocxx::instance instance;
+    mongocxx::uri uri("<connection string>");
+    mongocxx::client client(uri);
+
+    // start-db-coll
+    auto db = client["sample_restaurants"];
+    auto collection = db["restaurants"];
+    // end-db-coll
+
+    {
+        // replaces the fields and values of a document with the name "Bagels N Buns" with a document 
+        // with the same _id value and no fields other than the name "2 Bagels 2 Buns"
```

Comment:
S: you can shorten this comment; also, make sure it mentions the correct values
```suggestion
        // Replaces a document that has a "name" value of "Nobu" with a document that has a "name" of "La Bernadin"
```